--- options.c	2019-06-07 09:22:34.300748220 +0300
+++ options.c.change	2019-06-07 09:21:31.790009799 +0300
@@ -401,7 +401,7 @@
             exit(EXIT_FAILURE);
         }
 
-        if ((strlen(option.scpi_command) == 0) && (option.interactive == false))
+        if ((strlen(option.scpi_command) == 0) && (option.interactive == false) && (option.run_script == false))
         {
             error_printf("No SCPI command specified\n");
             exit(EXIT_FAILURE);
